compiler/luci/pass/src/ConvertNCHWToNHWCPass.test.cpp

```diff
@@ -696,6 +701,30 @@ TEST(ConvertNCHWToNHWC, Mean)
   EXPECT_EQ(2, new_rindices->at<loco::DataType::S32>(1));
 }
 
+TEST(ConvertNCHWToNHWC, Mean_keep_dims_false)
```

Comment:
Could you please add negative test that will fail with non 3d?
And maybe if possible, more for rejected conditions.